lib/java/logback.xml

```diff
@@ -15,10 +15,16 @@
     </encoder>
   </appender>
 
-  <appender name="FILE" class="ch.qos.logback.core.FileAppender">
-    <file>${logging.appender.file.fileName:-logback.log}</file>
-    <append>true</append>
-    <immediateFlush>true</immediateFlush>
+  <appender name="FILE" class="ch.qos.logback.core.rolling.RollingFileAppender">
+    <file>${logging.appender.file.fileName:-/tmp/logback-groovy.log}</file>
+    <rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
+    <fileNamePattern>/tmp/logback-groovy.%d{yyyy-MM-dd}.%i.log</fileNamePattern>
```

Comment:
Out of curiosity: wouldn't this break on Windows where there is no `/tmp` directory?